Header header
int32 ts
float32 vx
float32 vy
float32 vz
uint8 health_flag
